fluka/CMakeLists.txt

```diff
@@ -12,18 +12,18 @@ set(PUB_HEADERS fluka_funcs.h)
 set(LINK_LIBS)
 set(LINK_LIBS_STATIC)
 list(APPEND LINK_LIBS        ${FLUKA_DIR}/libflukahp.a)
-list(APPEND LINK_LIBS        dagmciface)
+list(APPEND LINK_LIBS        dagmc)
 list(APPEND LINK_LIBS        pyne_dagmc)
 list(APPEND LINK_LIBS        gfortran)
 list(APPEND LINK_LIBS_STATIC ${FLUKA_DIR}/libflukahp.a)
```

Comment:
again, having fully defined LINK_LIBS we set LINK_LIBS_STATIC, to be LINK_LIBS. In fact does it even have to be defined differently? Dont we just set the compile linkage later to be static, or arhive type for libs?